packages/@okta/vuepress-site/guides/sign-into-spa/section11/index.md

```diff
@@ -1,8 +1,19 @@
 ---
 title: Use the Access Token
 ---
-## Use the Access Token
 
-After your user is authenticated, your application has an access token in local storage that was issued by your Okta authorization server. You can use this token to authenticate requests made for protected resources on your server. 
+Your app will need to send requests to one or more APIs to perform actions or retrieve information.
```

Comment:
Your app needs to send requests to one or more APIs to perform actions or retrieve information.